phoenix-core-client/src/main/java/org/apache/phoenix/query/QueryConstants.java

```diff
@@ -640,4 +650,35 @@ enum JoinType {INNER, LEFT_OUTER}
             + SYSTEM_TASK_SPLIT_POLICY_CLASSNAME + "',\n" +
             TRANSACTIONAL + "=" + Boolean.FALSE + ",\n" +
             STORE_NULLS + "=" + Boolean.TRUE;
+
+    String CREATE_CDC_STREAM_STATUS_METADATA = "CREATE TABLE " + SYSTEM_CATALOG_SCHEMA + ".\"" +
+            SYSTEM_CDC_STREAM_STATUS_TABLE + "\"(\n" +
+            // PK columns
+            TABLE_NAME + " VARCHAR NOT NULL," +
+            STREAM_STATUS + " VARCHAR NOT NULL," +
+            // Non-PK columns
+            STREAM_NAME + " VARCHAR,\n" +
+            "CONSTRAINT " + SYSTEM_TABLE_PK_NAME + " PRIMARY KEY (" +
+            TABLE_NAME + "," + STREAM_STATUS + "))\n" +
+            HConstants.VERSIONS + "=%s,\n" +
+            ColumnFamilyDescriptorBuilder.KEEP_DELETED_CELLS + "=%s,\n" +
+            TRANSACTIONAL + "=" + Boolean.FALSE;
+
+    String CREATE_CDC_STREAM_METADATA = "CREATE TABLE " + SYSTEM_CATALOG_SCHEMA + ".\"" +
+            SYSTEM_CDC_STREAM_TABLE + "\"(\n" +
+            // PK columns
+            TABLE_NAME + " VARCHAR NOT NULL," +
+            STREAM_NAME + " VARCHAR NOT NULL," +
+            PARTITION_ID + " VARCHAR NOT NULL," +
+            // Non-PK columns
+            PARENT_PARTITION_ID + " VARCHAR," +
+            PARTITION_START_TIME + " BIGINT," +
```

Comment:
@virajjasani Does BIGINT make sense for start/end time since we can use (region_id) which is timestamp that we plan to use as sequence number? Or does it have to be Date as per the design doc?

Reply:
I see, in this case we can use LONG directly instead of DATE.